doc/builders/fetchers.chapter.md

```diff
@@ -82,6 +82,53 @@ Note that because the checksum is computed after applying these effects, using o
 
 Most other fetchers return a directory rather than a single file.
 
+
+## `fetchDebianPatch` {#fetchdebianpatch}
+
+A wrapper around `fetchpatch`, which takes:
+- `patch` and `hash`: the patch's filename without the `.patch` suffix,
+  and its hash after normalization by `fetchpatch` ;
+- `pname`: the Debian source package's name ;
+- `version`: the upstream version number ;
+- `debianRevision`: the [Debian revision number] if applicable ;
+- the `area` of the Debian archive: `main` (default), `contrib`, or `non-free`.
+
+Here is an example of `fetchDebianPatch` in action:
+
+```nix
+{ lib
+, fetchDebianPatch
+, buildPythonPackage
+}:
+
+buildPythonPackage rec {
+  pname = "pysimplesoap";
+  version = "1.16.2";
+  src = ...;
+
+  patches = [
+    (fetchDebianPatch {
+      inherit pname version;
+      debianRevision = "5";
+      name = "Add-quotes-to-SOAPAction-header-in-SoapClient";
+      hash = "sha256-xA8Wnrpr31H8wy3zHSNfezFNjUJt1HbSXn3qUMzeKc0=";
+    })
+  ];
+
+  ...
+}
+```
+
+Patches are fetched from `sources.debian.org`, and so must come from a
+package version that was uploaded to the Debian archive.  Packages may
+be removed from there once that specific version isn't in any suite
+anymore (stable, testing, unstable, etc.), so maintainers should use
+`copy-tarballs.pl` to archive the patch if it needs to be available
```

Comment:
This is wrong, `copy-tarballs` is automatically executed on a channel bump, so there's no need to do that manually.

Reply:
Turns out that this is only run against a very old NixOS: https://github.com/NixOS/nixos-org-configurations/blob/6008c5d9f653932d11a66d653f582d18591f76c4/modules/tarball-mirror.nix#L14

Which raises the question if it's not better to just vendor the patches here given that there are very few people only with access to said bucket.

Reply:
Vendoring all the patches is not OK because of the size of the repo

The version should just be updated to either unstable or stable and added to the release manager doc

Reply:
> Vendoring all the patches is not OK because of the size of the repo

I'd prefer a slightly larger tarball over unstable hashes. Also, the patches we're talking about are only a small fraction of all patches applied somewhere in nixpkgs. Finally, I wouldn't even expect the size of an average patch to matter much in comparison with to every checked in Cargo.lock, node-packages.nix, yarn.lock etc..

Reply:
Or we can just mirror the patches

Reply:
tarball-mirror